tests/Integration/SDK/SpanLimitsBuilderTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Integration\SDK\Trace;
+
+use AssertWell\PHPUnitGlobalState\EnvironmentVariables;
+use OpenTelemetry\SDK\Trace\SpanLimitsBuilder;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\SDK\Trace\SpanLimitsBuilder
```

Comment:
The `covers` annotation is also not really needed in the integration tests (and often the point of integration tests is to cover more than one class in conjunction)
We mainly use it in the unit tests to not have to create mocks for everything, because otherwise the code coverage report would make every code which is executed green, whether is expöicetly tested or not